src/ireiat/config.py

```diff
@@ -10,6 +10,7 @@
 
 # GIS-related parameters
 RADIUS_EARTH_MILES = 3958.8
+METERS_PER_MILE = 1609.34
 LATLONG_CRS = "EPSG:4326"
 ALBERS_CRS = "EPSG:5070"
```

Comment:
Strictly as an FYI:  I've read that ALBERS EPSG:9822 is very good for the continental US.  Lambert conic EPSG:9802 is also mentioned.  The one you have, EPSG:5070 also seems quite popular.